### PR TITLE
Commiting my files

### PR DESCRIPTION
**Pull request by Puja7629**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [x] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by Puja7629** [comment]
GitHub education file updates.

**Comment by Puja7629** [comment]
Updated again

**Comment by Puja7629** [comment]
Hey! @elisemoe can you please check it once again. I have filled the graduate shipping form last night according to IST. 
Please do let me know if you can't find it I will submit it once again.
Thank You!

**Comment by Puja7629** [comment]
Hey Education/Graduation.
I have filled the form last night according to IST.
Can you please check it once again.

Thanks & Regards
Puja Kumari.

File: _data/Puja7629/Puja7629.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Puja Kumari
```
Comment (juanpflores) [inline _data/Puja7629/Puja7629.md:1]: Don't forget to add `---` at the beginning and the end of this file.

**Comment by Puja7629** [comment]
Ok, Thanks for reviewing and mentioning my mistake in the file. I will update it and will make a new PR. Thanks

**Comment by Puja7629** [comment]
https://github.com/education/graduation/pull/4284   This is my new PR link. Should I fill the shipment form again?

**Comment by juanpflores** [comment]
It's not necessary to fill the form again

**Pull request closed without merging**